lms/views/admin/assignment.py

```diff
@@ -55,7 +55,7 @@ def assignment_dashboard(self):
                 "dashboard.assignment",
                 assignment_id=assignment.id,
                 _query={
-                    "public_id": assignment.course.application_instance.organization.public_id
+                    "public_id": assignment.course.application_instance.organization.public_id  # type: ignore
```

Comment:
This was already happening before it's now better typed.

Ignoring it in the admin pages.

Reply:
assignment.course is potentially None in the DB schema for historical data (old assignments, before we recorded the relationship to groupings)